css/obBase.css

```diff
@@ -202,13 +202,41 @@ form {
   width: calc(100% - 230px);
 }
 
-#pageNav {
+#appBar {
   position: fixed;
-  top: 0;
+  width: 100%;
+  z-index: 14;
+  background-color: #327EB8;
+  padding: 10px 8px;
```

Comment:
The app bar looks taller than in Mike's design, could the height be reduced to 29 or 28px from 35? Maybe drop the bottom padding.